scrapy/commands/__init__.py

```diff
@@ -104,3 +104,28 @@ def run(self, args, opts):
         Entry point for running commands
         """
         raise NotImplementedError
+
+
+class BaseRunSpiderCommands(ScrapyCommand):
```

Comment:
```suggestion
class BaseRunSpiderCommand(ScrapyCommand):
```
And the corresponding updates in child classes.